### PR TITLE
Update test application to Sylius 1.10 and fix default configuration

### PR DESCRIPTION
**Pull request by delyriand**
Require #13

### CODE REVIEW

File: src/EventListener/DisableFirewallListener.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        $this->channelContext = $channelContext;
+    }
+
+    public function __invoke(RequestEvent $event): void
+    {
+        if (!$event->isMasterRequest()) {
+            return;
+        }
+        $request = $event->getRequest();
+        if (\in_array($request->attributes->get('_route'), ['_wdt', '_profiler', '_profiler_search', '_profiler_search_results'], true)) {
```
Comment (lanfisis) [inline src/EventListener/DisableFirewallListener.php]: Put value into a const

File: src/EventListener/DisableFirewallListener.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            // allow profiler routes
+            return;
+        }
+
+        $currentChannel = $this->channelContext->getChannel();
```
Comment (lanfisis) [inline src/EventListener/DisableFirewallListener.php]: Beware because this line could throw an exception if no channel are found

**Review by lanfisis — CHANGES_REQUESTED** [review]

File: tests/Application/config/routes/sylius_admin_api.yaml [hunk 3]
```diff
@@ -1,3 +0,0 @@
-sylius_admin_api:
```
Comment (maximehuran) [inline tests/Application/config/routes/sylius_admin_api.yaml:1]: This file has no symbolic link ?

File: tests/Application/config/packages/_sylius.yaml [hunk 4]
```diff
@@ -2,12 +2,9 @@ imports:
     - { resource: "@SyliusCoreBundle/Resources/config/app/config.yml" }
 
     - { resource: "@SyliusAdminBundle/Resources/config/app/config.yml" }
-    - { resource: "@SyliusAdminApiBundle/Resources/config/app/config.yml" }
 
     - { resource: "@SyliusShopBundle/Resources/config/app/config.yml" }
 
-#    - { resource: "@SyliusApiBundle/Resources/config/app/config.yaml" }
```
Comment (maximehuran) [inline tests/Application/config/packages/_sylius.yaml:9]: We do not have APIs ?

File: src/EventListener/DisableFirewallListener.php (hunk 1 above)
Comment (maximehuran) [reply src/EventListener/DisableFirewallListener.php:45]: Why do you want to put it in conf ?

File: tests/Application/config/routes/sylius_admin_api.yaml (hunk 3 above)
Comment (delyriand) [reply tests/Application/config/routes/sylius_admin_api.yaml:1]: this file was [removed in 1.10](https://github.com/Sylius/Sylius/commit/5f3d5a8273cc4639b40037a135259c4b5ac1bada#diff-01734a8d027dc63fd22c87d8937ad428f3ab3798a489d32dc0a22fbed63395b8)

File: tests/Application/config/packages/_sylius.yaml (hunk 4 above)
Comment (jacquesbh) [reply tests/Application/config/packages/_sylius.yaml:9]: Removed in 1.10 as well.

**Review by jacquesbh — APPROVED** [review]

**Review by lanfisis — APPROVED** [review]

**Comment by lanfisis** [comment]
Thank you 🙏

**Pull request merged**